modules/aws_vm/main.tf

```diff
@@ -124,3 +124,11 @@ resource "aws_ebs_volume" "raid_disk" {
   tags              = merge(var.tags, tomap({ Name : "${var.name}-vm" }))
   encrypted         = var.enable_ebs_encryption
 }
+
+# Reference the feature flag variable name, an example reference to suppress TFLint warning
```

Comment:
Is this going to always echo this line even for standard customers who are not going to use the flag? You could use `count` so it's only echoed when the flag is used.  Just a suggestion not a required change.

Reply:
@jarpat, the local-exec generates console output during the plan or apply command that looks like this:

```
module.jump[0].terraform_data.example (local-exec): Executing: ["/bin/sh" "-c" "echo The enable_nist_features flag value is: false"]
module.jump[0].terraform_data.example (local-exec): The enable_nist_features flag value is: false
```
I have also added an output to ouputs.tf so the inline expression that shows in the console for the local-exec output is pretty much just showing the value of the flag, whether it is true or false. Since the default value is false if not present in terraform-input.tfvars, The  output would always be present.

Reply:
Yeah I'm with this thread being resolved, forgot that output will always be present as well.